client/settings/fraud-protection/index.tsx

```diff
@@ -13,20 +13,22 @@ import { ProtectionLevels } from './components';
 import './style.scss';
 import FraudProtectionTour from './tour';
 
-const FraudProtection = () => {
+const FraudProtection: React.FC = () => {
 	return (
 		<>
 			<Card className="fraud-protection">
 				<CardBody>
-					<h4>
-						<span id="fraud-protection-card-title">
-							{ __(
-								'Set your payment risk level',
-								'woocommerce-payments'
-							) }
-						</span>
-					</h4>
-					<ProtectionLevels />
+					<>
```

Comment:
Why did we need this fragment wrapper here?

Reply:
We need a fragment wrapper because `<Card />` and `<FraudProtectionTour />` are on the same level.

Reply:
I thought you were referring to the outer fragment at first.